src/commands/init.js

```diff
@@ -146,6 +147,66 @@ async function init() {
 const sleep = (time: number = 1000) =>
   new Promise(resolve => setTimeout(resolve, time));
 
+const getRunScript = (scriptName: string) => {
+  const runCommand = scriptName === 'start' ? 'yarn' : 'yarn run';
+  return `${runCommand} ${scriptName}`;
+};
+
+/**
+ * Adds Haul to package.json scripts
+ */
+const addToPackageScripts = async (cwd: string) => {
+  const pjson = JSON.parse(
+    fs.readFileSync(path.join(cwd, 'package.json')).toString(),
+  );
+
+  const scripts = pjson.scripts || {};
+
+  const haulScript = Object.keys(scripts).find(name =>
+    scripts[name].includes('haul'));
+
+  if (haulScript) {
+    ora().info(
+      `Haul is already in your package's scripts. You can start it by running '${getRunScript(haulScript)}'`,
```

Comment:
do you mean in `package.json`? If so I'd just say:
Haul already exists in your package.json. Start Haul by running...